src/CONST.ts

```diff
@@ -1252,6 +1252,7 @@ const CONST = {
     },
     CENTRAL_PANE_ANIMATION_HEIGHT: 200,
     LHN_SKELETON_VIEW_ITEM_HEIGHT: 64,
+    SEARCH_SKELETON_VIEW_ITEM_HEIGHT: 90,
```

Comment:
Based on what I see in Figma, on mobile these are 100px tall:
![CleanShot 2024-06-13 at 16 38 07@2x](https://github.com/Expensify/App/assets/2319350/fa9f2406-dcbf-4f46-90f0-ebfc74eb5e37)

Reply:
+1

Reply:
I updated. Here is the result:

<img width="355" alt="Screenshot 2024-06-14 at 18 34 22" src="https://github.com/Expensify/App/assets/170341821/752b05f4-1c8d-4d30-9a14-5fded1c66096">